manifests/profile/mistral.pp

```diff
@@ -369,6 +391,14 @@
           content => template('st2/etc/systemd/system/mistral.service.erb'),
           notify  => Service['mistral'],
         }
+        file { '/etc/systemd/system/mistral-api.service':
+          ensure => file,
+          owner  => 'root',
+          group  => 'root',
+          mode   => '0444',
+          content => template('st2/etc/systemd/system//mistral-api.service.erb'),
```

Comment:
this *shouldn't* cause issue, but I cannot say for sure. Watch the double slash.